packages/core/admin/admin/src/components/GuidedTour/Tours.tsx

```diff
@@ -255,32 +255,51 @@ const tours = {
     {
       name: 'Introduction',
       content: ({ Step }) => (
-        <Step.Root sideOffset={-36} withArrow={false}>
-          <Step.Title id="tours.apiTokens.Introduction.title" defaultMessage="API tokens" />
+        <Step.Root side="top" sideOffset={32} withArrow={false}>
+          <Step.Title
+            id="tours.apiTokens.Introduction.title"
+            defaultMessage="Last but not least, API tokens"
+          />
           <Step.Content
             id="tours.apiTokens.Introduction.content"
-            defaultMessage="Create and manage API tokens with highly customizable permissions."
+            defaultMessage="Create and manage API tokens with highly customizable permissions to control access to your content."
```

Comment:
Maybe say _API access_? Otherwise it's confusing how this fits with RBAC

Reply:
The "Create and" should be gone too. Sounds good to me. What do you think of: `Control API access with highly customizable permissions`? The title already mentions API tokens.